build/teamcity-assert-clean.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+export BUILDER_HIDE_GOPATH_SRC=1
+
+build/builder.sh /bin/bash -c '! git status --porcelain | read || (git status; git diff -a 1>&2; exit 1)'
+
```

Comment:
I was planning to call it in a post-build step in every builder, not just the check build.